di_website/visualisation/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.shortcuts import redirect
+
+from wagtail.core.models import Page
+from wagtail.core.fields import StreamField
+from wagtail.admin.edit_handlers import StreamFieldPanel
+
+from .blocks import ChartBlock
+
+
+class VisualisationsPage(Page):
+    """
+    Parent page for all visualisations
+    """
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['visualisation.ChartPage']
+
```

Comment:
@edwinmp 

As you only want one of these pages as a chart parent/folder, I would add this line to make the page unique:

```
max_count = 1
```

See: 
https://docs.wagtail.io/en/v2.9/reference/pages/model_reference.html#wagtail.core.models.Page.max_count